examples/transient_adiabatic_mg/c5g7-td-2d-mg.py

```diff
@@ -0,0 +1,142 @@
+import numpy as np
+import os
+import openmc
+import openmc.mgxs
+import openmc.kinetics as kinetics
+
+from geometry_2d_mg import materials, surfaces, universes, cells, lattices, geometry, mgxs_lib_file
+from mgxs_lib import mgxs_data
+materials_file = openmc.Materials(geometry.get_all_materials().values())
+os.environ['OPENMC_MG_CROSS_SECTIONS'] = 'mgxs.h5'
```

Comment:
Does the environment variable definitely have to be set? That seems like maybe something the user should be able to set. For instance, if someone set this to a different value, they wouldn't want it changed. Why not take the approach used in `pincell_multigroup/build_xml.py`?